spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkWrite.java

```diff
@@ -742,8 +742,8 @@ private PartitionedDataWriter(
         Schema dataSchema,
         StructType dataSparkType,
         long targetFileSize,
-        boolean fanoutEnabled) {
-      if (fanoutEnabled) {
+        boolean fanout) {
+      if (fanout) {
```

Comment:
Why rename this? It seems like `fanoutEnabled` was fine in context. I don't think it implied that a broader option was set.

Reply:
It is a leftover from the previous version where enabling the fanout writer did not mean using it. I can remove this one.